docs/reference/commands/query_syntax/explain.md

```diff
@@ -0,0 +1,53 @@
+---
+title: EXPLAIN
+order: 0
+---
+
+# EXPLAIN
+
+`EXPLAIN` statements provide information about how a query will be executed, showing the query plan that the database will use.
```

Comment:
Change this to indicate EXPLAIN shows the unoptimized, optimized, and physical plans for the query.